src/components/ListDetail/ListDetail.tsx

```diff
@@ -193,33 +201,44 @@ export const ListDetail: React.FC<Props> = ({
     }
   }
 
+  const childrenWithProps = React.Children.map(
+    listDetailItems(),
+    (child: any, i: any) => {
+      // Checking isValidElement is the safe way and avoids a typescript
+      // error too.
+      if (React.isValidElement(child)) {
+        return React.cloneElement<Props>(child, {
+          id: idVar[i],
+          ariaLabelledBy: ariaLabelledByVar[i],
+        });
+      }
+      return child;
+    },
+  );
 
-  const childrenWithProps = React.Children.map(listDetailItems(), (child: any, i: any) => {
-    // Checking isValidElement is the safe way and avoids a typescript
-    // error too.
-    if (React.isValidElement(child)) {
-      return React.cloneElement<Props>(child, {
-        id: idVar[i],
-        ariaLabelledBy: ariaLabelledByVar[i],
-      });
-    }
-    return child;
-  });
-
-  const componentClassName = clsx(styles['list-detail'], className, {
-  });
+  const componentClassName = clsx(styles['list-detail'], className, {});
 
   return (
     <div className={componentClassName} {...other}>
       <div className={styles['list-detail__header']}>
-        <ul className={styles['list-detail__list']} role="tablist">
+        <ul
+          className={clsx(styles['list-detail__list'], {
+            [styles['list-detail__list--sequence']]: variant === 'sequence',
+          })}
+          role="tablist"
+        >
           {listDetailItems().map((tab: any, i: any) => {
             const isActive = activeIndexState === i;
+            const itemVariant = variant && tab.props.variant;
```

Comment:
@csnizik What was your reasoning around changing this to a `const`? Just curious if it was more straightforward of a name or something. Thanks!

Reply:
Did that to avoid having to add another condition to the logic for classNames; I originally had double conditionals on the li classNames to check if variant was sequence and if tab.props.variant had a value, which looked awkward. I used this way so that `itemVariant` will only hold a value if both conditions are true (list === sequence && this tab's variant is set)